setup.py

```diff
@@ -17,6 +17,7 @@
         "reportportal-client",
         "robotframework>=4",
         "robotframework-debuglibrary",
+        "robotframework-lsp",
```

Comment:
this can be moved to requirements-dev.txt as its required for development work but not for regular sanity/regressions.